var/spack/repos/builtin/packages/hpctoolkit/package.py

```diff
@@ -27,6 +27,8 @@ class Hpctoolkit(AutotoolsPackage):
     test_requires_compiler = True
 
     version("develop", branch="develop")
+    version("2023.08.stable", branch="release/2023.08")
+    version("2023.08.1", tag="2023.08.1")
```

Comment:
Can you add a commit reference too?

Reply:
@alalazo   I'm not sure I understand.
Do you want both `tag="..."` and `commit="..."` or just commit?

Reply:
Both is fine. You can look at `mpl` or `axom` as an example.

Reply:
Ok, done.